content/docs/guides/auto-suspend-guide.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Configuring Auto-suspend for Neon computes
-subtitle: Learn how to configure Neon's Auto-suspend feature to control when your
+title: Configuring Autosuspend for Neon computes
+subtitle: Learn how to configure Neon's Autosuspend feature to control when your
   compute scales to zero
 enableTableOfContents: true
 updatedOn: '2023-10-24T18:56:54.982Z'
 ---
 
-Neon's [Auto-suspend](/docs/introduction/auto-suspend) feature controls when a Neon compute instance transitions to an `Idle` state (scales to zero) due to inactivity. This guide demonstrates how to configure the _Auto-suspend_ setting in your Neon project.
+Neon's [Autosuspend](/docs/introduction/auto-suspend) feature controls when a Neon compute instance transitions to an `Idle` state (scales to zero) due to inactivity. This guide demonstrates how to configure the autosuspend setting in your Neon project.
 
-[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating new compute endpoints.
+[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the autosuspend setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the Autosuspend setting for a new or existing project sets the project's default autosuspend setting, which is used from that point forward when creating new compute endpoints.
```

Comment:
```suggestion
[Neon Pro Plan](/docs/introduction/pro-plan) users can configure the autosuspend setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the autosuspend setting for a new or existing project sets the project's default autosuspend setting, which is used from that point forward when creating new compute endpoints.
```

Reply:
I think this 2n should be lowercase as well - but disregard if I've got it wrong